os/tools/mkbinheader.py

```diff
@@ -16,6 +16,9 @@
 # language governing permissions and limitations under the License.
 #
 ############################################################################
+
+from __future__ import print_function
+from optparse import OptionParser
```

Comment:
Sorry for the confusion...
`from optparse import OptionParser` is not needed...